src/test/commands.test.ts

```diff
@@ -0,0 +1,126 @@
+/* 
+ eslint-disable 
+  @typescript-eslint/no-unsafe-assignment,
+  @typescript-eslint/no-explicit-any, 
+  @typescript-eslint/no-unsafe-member-access 
+*/
+
+import * as assert from 'assert';
+import * as fs from 'fs/promises';
+import * as os from 'os';
+import * as path from 'path';
+import * as util from '../util';
+import * as vscode from 'vscode';
+import { ImandraxLanguageClient } from '../imandrax_language_client/imandrax_language_client';
+
+
+suite('Commands, Simple Test Suite', () => {
+  suiteTeardown(() => {
```

Comment:
This is great, thanks for setting up the whole test framework, this will improve our lives forever!